RecoLocalCalo/HGCalRecAlgos/python/hgcalRecHitToolsPartialWafer_cff.py

```diff
@@ -0,0 +1,19 @@
+import FWCore.ParameterSet.Config as cms
+
+from RecoLocalCalo.HGCalRecAlgos.hgcalCheckToolDigiEE_cfi import *
+
+hgcalCheckToolDigiHE = hgcalCheckToolDigiEE.clone(
+    source = cms.InputTag('simHGCalUnsuppressedDigis', 'HEfront'),
+    nameSense  = "HGCalHESiliconSensitive",
```

Comment:
Why is this file in \python, rather than in the \test folder? It refers to a plugin in \test, it should stay there

```suggestion
    source = ('simHGCalUnsuppressedDigis', 'HEfront'),
    nameSense  = "HGCalHESiliconSensitive",
```